data/json/itemgroups/military.json

```diff
@@ -1,4 +1,89 @@
 [
+  {
+    "type": "item_group",
+    "id": "military_standard_assault_rifles",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "m4a1", "prob": 88, "charges": [ 0, 30 ] },
+      { "item": "m27iar", "prob": 10, "charges": [ 0, 30 ] },
+      { "item": "m16a4", "prob": 2, "charges": [ 0, 30 ] }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "military_standard_shotguns",
+    "subtype": "distribution",
+    "entries": [ { "item": "m1014", "prob": 60, "charges": [ 0, 8 ] }, { "item": "mossberg_590", "prob": 40, "charges": [ 0, 9 ] } ]
+  },
+  {
+    "type": "item_group",
+    "id": "military_standard_sniper_rifles",
+    "subtype": "distribution",
+    "entries": [ { "item": "m2010", "prob": 20, "charges": [ 0, 5 ] }, { "item": "m110a1", "prob": 80, "charges": [ 0, 20 ] } ]
+  },
+  {
+    "type": "item_group",
+    "id": "military_standard_lmgs",
+    "subtype": "distribution",
+    "entries": [ { "item": "m249", "prob": 100, "charges": [ 0, 200 ] } ]
+  },
+  {
+    "type": "item_group",
+    "id": "military_standard_pistols",
+    "subtype": "distribution",
+    "entries": [
+      { "item": "m17", "prob": 95, "charges": [ 0, 17 ] },
+      { "item": "glock_19", "prob": 3, "charges": [ 0, 15 ] },
+      { "item": "m1911_MEU", "prob": 2, "charges": [ 0, 7 ] }
```

Comment:
Where's the M9? That's the direct predecessor to the M17, which aren't planned to be fully delivered until 2027(https://www.armytimes.com/news/your-army/2017/01/20/11-things-you-need-to-know-about-the-army-s-new-handgun/). 😕